packages/edit-widgets/src/components/more-menu/tools-more-menu-group.js

```diff
@@ -14,7 +9,7 @@ const { Fill: ToolsMoreMenuGroup, Slot } = createSlotFill(
 
 ToolsMoreMenuGroup.Slot = ( { fillProps } ) => (
 	<Slot fillProps={ fillProps }>
-		{ ( fills ) => ! isEmpty( fills ) && fills }
+		{ ( fills ) => fills.length > 0 && fills }
```

Comment:
Can `fills` be undefined?

Reply:
Thanks for double-checking, but nope, it's always an array:

https://github.com/WordPress/gutenberg/blob/b33b2564f8be2276fdae5411c3338fec517a7241/packages/components/src/slot-fill/slot.js#L70-L92

Reply:
Well, then I don't understand why we needed the second `fills` here 😅? It probably would be safe also to delete it.

Reply:
> Well, then I don't understand why we needed the second `fills` here 😅? It probably would be safe also to delete it.

I left it as-is, but I can open a follow-up PR if you feel strongly about it 😅